composer.json

```diff
@@ -33,6 +33,9 @@
         "symfony/console":  "^3.0 || ^4.0",
         "webmozart/glob":   "^4.1"
     },
+    "conflict": {
+        "symfony/console": "3.4.16|4.1.5"
```

Comment:
Better simply bump `^3.4.17 | ^4.1.6` above instead - no need to support older versions, as they will obviously have older bugs anyway.

Reply:
Agreed, but they've not been released yet

Reply:
Then let's wait for a release and bump this afterwards

Reply:
Note that this affects everything that uses `symfony/console`, so doing a hunt for downstream consumers is like fighting windmills right now :-\

Reply:
Ok I've pushed your suggested change now

Reply:
I'll restart the build at the next symfony releases :+1: